app/views/spam2/_users.html.erb

```diff
@@ -71,7 +71,7 @@ $(document).ready(function () {
           <tr id="n<%= user.id %>">
             <td><input  class="selectedId" value="<%= user.uid %>" type="checkbox" /></td>
             <td>
-              <i class="fa fa-user text-<% if user.status == 0 %>danger<% elsif user.role == "admin" || user.role == "moderator" %>primary <% else %>secondary<%end%>"></i> 
+			  <i class="fa fa-user text-<% if user.status == 0 %>danger<% elsif user.can_moderate? %>primary <% else %>secondary<%end%>"></i>
```

Comment:
Hi @simranbiswas. Could you please remove the extra spaces and retain the earlier indentation?

Reply:
Yeah will do it in a few minutes

Reply:
Can you check the latest commit and see if the indentation is proper or not? Actually my text editor somehow wasn't showing any extra space and editing directly on github was a bit confusing that is why so many commits, many apologies!